### ISSUE TITLE
cannot access custom kv store through the UI

### ISSUE THREAD

**Issue by bacongobbler** [radu-matei/spin-kv-explorer#26]
Given the following spin.toml:

```
spin_manifest_version = 2

[application]
name = "foo"
version = "0.1.0"

[application.trigger.http]
base = "/"

[[trigger.http]]
id = "trigger-kv-explorer"
component = "kv-explorer"
route = "/internal/kv-explorer/..."

[component.kv-explorer]
source = { url = "https://github.com/fermyon/spin-kv-explorer/releases/download/v0.9.0/spin-kv-explorer.wasm", digest = "sha256:07f5f0b8514c14ae5830af0f21674fd28befee33cd7ca58bc0a68103829f2f9c" }
allowed_outbound_hosts = ["redis://*:*", "mysql://*:*", "postgres://*:*"]
key_value_stores = ["default", "foo"]
```

And the following runtime-config.toml

```
[key_value_store.foo]
type = "spin"
path = ".spin/foo.db"
```

When I run `spin up`:

```
><> spin up --runtime-config-file runtime-config.toml
Logging component stdio to ".spin/logs/"
Storing default key-value data to ".spin/sqlite_key_value.db"

Serving http://127.0.0.1:3000
Available Routes:
  kv-explorer: http://127.0.0.1:3000/internal/kv-explorer (wildcard)
```

I expect that I should be able to interact with both the default KV store as well as the custom "foo" KV store. However it appears that I can only interact with the default KV store, and there is no way for me to switch views to look at the contents within "foo".

![image](https://github.com/fermyon/spin-kv-explorer/assets/1360539/cd40cfd0-4872-46be-a2b5-5980b484f1ad)

**Comment by bacongobbler** [comment]
This is listed as a known limitation [in the README](https://github.com/fermyon/spin-kv-explorer#known-limitations):

> the explorer can only be used with Spin's default key/value store. When this will be configurable, this component will support working with custom stores as well.

It looks like [there is API support](https://github.com/fermyon/spin-kv-explorer/blob/1f3da02c936e8417150936c8a24ad9b0089d520f/explorer/main.go#L106-L117) for interacting with custom KV stores. There's just no way to do it through the provided UI.

**Comment by mikkelhegn** [comment]
Implemented in #27
